gateway/README.md

```diff
@@ -2,116 +2,167 @@
 
 ## Overview
 
-This basic example demonstrates how to expose a service or a function through an API in a public or secure manner.
+This basic example demonstrates how to expose a service or a function through an API in a public or secure manner through the console UI or manually using kubectl.
 
 ## Prerequisites
 
 - Kyma as the target deployment environment.
-- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.9.0
-- An environment to which to deploy the example.
+- An environment to which you deploy the example.
 
 ## Installation
 
-First of all, export your environment as variable by replacing the `<environment>` placeholder in the following command and running it.
+This section contains installation steps on how to expose a service or a function through the console UI, and also manually, using kubectl.
+
+### Exposure through the console UI
+
+#### Create a service
+
+1. Open the Kyma console (https://console.kyma.local/) and choose or create the Environment in which you want to deploy the example.
+2. Click the **Deploy new resource to the environment** button, select the `deployment.yaml` file from the `lambda` or `service` directory in this example, and click **Upload**.
+
+#### Expose a service without authentication
+
+1. Select the **Services** button, click on the name of the service you have created. The name should be the same as the Service or Function name in the `deployment.yaml` file.
```

Comment:
`1. Select the **Services** button and click on the name of the service you have created.`